src/main/java/gift/dto/PurchaseOptionRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package gift.dto;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record PurchaseOptionRequestDto(
+        @NotNull(message = "수량을 입력해주세요")
+        @Positive(message = "수량은 음수가 될 수 없습니다")
+        @Max(value = 100000000, message = "수량은 1억개를 넘을 수 없습니다")
```

Comment:
```suggestion
        @Max(value = 100_000_000, message = "수량은 1억개를 넘을 수 없습니다")
```
이렇게 쓰면 가독성이 더 좋아질 것 같네요😊

Reply:
dto 전체에 대해 확인 후 구분자 반영하겠습니다.